tests/01-jwt.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { chai } from 'mochainon';
```

Comment:
Yes, I want the jwt and api-key tests to run first. So if sth breaks within them, the error will show in the right place and not in the auth tests

Reply:
I'm not sure what this means.

Reply:
The `auth` spec relies on the underlying `jwt` and `api-key` modules. If we ever break something in those modules, it's very likely that tests on `auth` spec will also break. In that case, I want to make sure they break first on the "low-level" modules' spec. It's not that relevant because we are not bailing on the first error but still.

TL;DR: In my opinion the ordering of tests here matter and I want those 2 spec modules to run before the `auth` one

Reply:
I don't really agree with this - test order really shouldn't ever matter, and it's not what we do everywhere else - but I don't care very much, and I'm totally happy to ignore it :smiley: